pkg/synchronization/controller.go

```diff
@@ -54,7 +55,9 @@ const (
 	// see https://github.com/jtblin/kube2iam#namespace-restrictions
 	allowedRolesAnnotation = "iam.amazonaws.com/allowed-roles"
 
-	maxSyncWorkers = 10
+	maxSyncWorkers             = 10
+	defaultPagerdutyGeneric    = "5d11612f25b840faaf77422edeff9c76"
+	defaultPagerdutyCloudwatch = "https://events.pagerduty.com/adapter/cloudwatch_sns/v1/124e0f010f214a9b9f30b768e7b18e69"
```

Comment:
These defaults were in the `buildNotifcations` function.

Reply:
These values should come from configuration, not be constants. Please move them.

Reply:
I'll do this in a follow up PR as this PR is big enough.